llvm/tools/llvm-exegesis/lib/CodeTemplate.cpp

```diff
@@ -57,6 +57,15 @@ const MCOperand &InstructionTemplate::getValueFor(const Operand &Op) const {
   return getValueFor(Instr->Variables[Op.getVariableIndex()]);
 }
 
+MCOperand &InstructionTemplate::getValueForOperandIdx(unsigned OpIdx) {
```

Comment:
These are no longer used in the current patch

Reply:
Addressed

Reply:
Addressed